packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -29,6 +30,51 @@ const releaseActionController = {
     };
   },
 
+  async createMany(ctx: Koa.Context) {
+    const releaseId: CreateManyReleaseActions.Request['params']['releaseId'] = ctx.params.releaseId;
+    const releaseActionsArgs: CreateManyReleaseActions.Request['body'] = ctx.request.body;
+
+    await Promise.all(
+      releaseActionsArgs.map((releaseActionArgs) => validateReleaseAction(releaseActionArgs))
+    );
+
+    const releaseService = getService('release', { strapi });
+
+    const releaseActions = await strapi.db.transaction(async () => {
+      const releaseActions = await Promise.all(
+        releaseActionsArgs.map(async (releaseActionArgs) => {
+          try {
+            const action = await releaseService.createAction(releaseId, releaseActionArgs);
+
+            return action;
+          } catch (error) {
+            if (
+              error instanceof errors.ValidationError &&
+              error.message ===
+                `Entry with id ${releaseActionArgs.entry.id} and contentType ${releaseActionArgs.entry.contentType} already exists in release with id ${releaseId}`
+            ) {
+              return null;
+            }
+
+            throw error;
+          }
+        })
+      );
+
+      return releaseActions;
+    });
```

Comment:
Was toying around, and not sure if possible, but having the custom error might help, wdyt?

```suggestion
const createActionIfNotExists = async (args) =>
  releaseService
    // Try to create the action
    .createAction(releaseId, args)
    // Ignore if action already exists
    .catch((error) => {
      if (error instanceof ActionAlreadyExists) {
        return null;
      }
      throw error;
    });

const releaseActions = strapi.db.transaction(() =>
  async.map(releaseActionsArgs, createActionIfNotExists)
);

```

Reply:
Definitely sounds better, let me check how can I create the custom error